datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -278,7 +279,7 @@ public DataCapturingService(@NonNull final Context context, @NonNull final Strin
             throw new SetupException("Android connectivity manager is not available!");
         }
         surveyor = new WiFiSurveyor(context, connectivityManager, authority, accountType);
-        fromServiceMessageHandler = new FromServiceMessageHandler(context, this);
+        fromServiceMessageHandler = new FromServiceMessageHandler(context.getMainLooper(), context, this);
```

Comment:
You are getting the looper from the context. So why do you not just call `context.getMainLooper()` inside the super call? That way you would not need to change this line.

Reply:
Good point, I reverted this API change.